content/tutorials/stack-outputs-and-references/index.md

```diff
@@ -0,0 +1,709 @@
+---
+title: "Stack Outputs and References"
+title_tag: "Stack Outputs and References"
+layout: single
+
+# A succinct description of the tutorial. It appears on the Tutorials home and collection pages.
+description: Learn more about exporting and referencing stack outputs in Pulumi.
+
+# A similar description used for search results and social-media previews.
+meta_desc: Learn more about exporting and referencing stack outputs in Pulumi.
+
+# An image for the tutorial. It appears on tutorial page and in social-media previews.
+meta_image: meta.png
+
+# An optional video for the tutorial. When present, it appears at the top of the page, replacing
+# the meta image. YouTube and HTML5 video sources are supported.
+video:
+    youtube: S-KfMsh78H8
+
+# The order in which the tutorial appears in most lists. Order is ascending, so higher numbers
+# mean the tutorial will appear further down the list. Positive integers only.
+weight: 999
+
+# A brief summary of the tutorial. It appears at the top of the tutorial page. Markdown is fine.
+summary: |
+    In this tutorial, you will learn how to work with stack outputs, specifically how to export values from a stack and how to reference those values from another stack. You will do this by creating a simple AWS Lambda Function that will write a file to an S3 bucket. You will also create an EventBridge Scheduler resource in a new stack that will run the Lambda function from the first stack on a scheduled basis.
+
+# A list of three to five things the reader will have learned by the end of the tutorial.
+youll_learn:
+    - How to create a stack output
+    - How to view the stack output
+    - How to reference the output from a different stack
+
+# A list of tutorial prerequisites. Markdown is fine. Keep it simple; no need to be exhaustive here.
+prereqs:
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup) and [access token](/docs/pulumi-cloud/accounts/#access-tokens)
+    - The [Pulumi CLI](/docs/install/)
+    - An [Amazon Web Services](https://aws.amazon.com/) account
+    - The [AWS CLI](https://aws.amazon.com/cli/)
+    - Install [NodeJS](/docs/languages-sdks/javascript/) or [Python](/docs/languages-sdks/python/)
```

Comment:
```suggestion
    - [Node.js](/docs/languages-sdks/javascript/) or [Python](/docs/languages-sdks/python/) installed
```

Reply:
I put the Install word first because it doesn't like when it starts with a bracket:

<img width="598" alt="image" src="https://github.com/user-attachments/assets/d7da4048-5708-4147-89f6-740aa60743a4">